paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/DropdownFieldController.kt

```diff
@@ -20,6 +21,11 @@ internal class DropdownFieldController(
     override val errorMessage: Flow<Int?> = MutableStateFlow(null)
     override val isComplete: Flow<Boolean> = MutableStateFlow(true)
 
+    override val elementType: ElementType = when (config) {
+        is IdealBankConfig -> ElementType.IdealBank
+        is CountryConfig -> ElementType.Country
+    }
```

Comment:
Similarly, `ElementType` can be a property of `DropdownConfig`, and `override val elementType = config.elementType`. Same for `TextFieldController`.

Reply:
yes, I like it!